src/wizards/tokenSwapDealWizard/wizardManager.ts

```diff
@@ -88,6 +88,10 @@ export class WizardManager {
     // if we are accessing an already existing deal, get its registration data
     this.registrationData = params.id ? await this.getDeal(params.id) : new DealRegistrationTokenSwap();
 
+    if (wizardType === WizardType.makeAnOffer || wizardType === WizardType.partneredDeal) {
```

Comment:
I don't understand why `wizardType === WizardType.partneredDeal`?  Can be documented?

In any case, needs to be on the clone, not the original.

Reply:
It's because when creating new DealRegistrationTokenSwap() it has partnerDAO set to undefined. We need to change it to an actual `IDAO` object, otherwise partnerDao stage is broken